ProcessLib/ComponentTransport/ComponentTransportProcess.h

```diff
@@ -146,7 +146,7 @@ class ComponentTransportProcess final : public Process
         _local_assemblers;
 
     /// Solutions of the previous time step
-    std::array<std::unique_ptr<GlobalVector>, 2> _xs_previous_timestep;
+    std::vector<std::unique_ptr<GlobalVector>> _xs_previous_timestep;
```

Comment:
Is this a bug fix?

Reply:
Last pull request (#2341) achieved functionality of staggered coupling scheme in single component transport process which has two primary variables, i.e., one for hydraulic pressure and the other for component concentration. Currently, we would like to use staggered scheme for multi-component transport simulation.